engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-advanced-sync-options.html

```diff
@@ -0,0 +1,54 @@
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="standard" />
+      <span translate>
+        Optimized Sync
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Standard sync, optimized for speed by bypassing any unneeded steps.
```

Comment:
I tend to prefer to leave nothing selected by default as to make the user make the selection.  If they've arrive at this page they most likely don't even want a standard sync as they could have selected it from the bulk drop down. Its here more for explanation purposes.